source/includes/fact-verifier-unsupported.rst

```diff
@@ -0,0 +1,18 @@
+The verifier doesn't check the following namespaces: 
+
+- Sharded collections
+- Capped collections
+- Collections with TTL indexes
+- Collections that don't use the default collation
+- Views
+
+The verifier doesn't check the following collection features:
+
+- Collection metadata
+- Sharding configurations
+- Indexes
+
+To ensure a successful migration, script additional checks for these
```

Comment:
"To ensure a successful migration" -> "To verify the above data and metadata"

Reply:
Fixed